aws/service_v2.go

```diff
@@ -131,6 +157,106 @@ func DynamoDbClient(ctx context.Context, d *plugin.QueryData) (*dynamodb.Client,
 	return svc, nil
 }
 
+// Ec2Client returns the service client for AWS Ec2 service
+func Ec2Client(ctx context.Context, d *plugin.QueryData) (*ec2.Client, error) {
```

Comment:
```suggestion
func EC2Client(ctx context.Context, d *plugin.QueryData) (*ec2.Client, error) {
```

Reply:
Please also remember to update comments, logging statements, etc. that reference this function name